### ISSUE TITLE
Use Inlite theme with keyboard only

### ISSUE THREAD

**Issue by alecpl** [tinymce/tinymce#3169]
The idea behind this new theme is interesting, however I found it problematic to use with keyboard only. E.g.:
1. When I select some text a toolbar appears, but how do I move focus to the toolbar using only keyboard? The same for other toolbars.
2. Pressing Tab to leave the editor does not hide the visible toolbar. If I leave the editor with Tab key a toolbar should disappear.
3. When I'm creating a table and cursor is inside of the table, how do I escape the table (move cursor outside of the table)? Tab key inside the table creates a new row, should it leave the table?

**Comment by spocke** [comment]
Good questions.

1) Keyboard focus can be done using Alt+F10 however this doesn't work on linux so we will probably add an alternative F10 as well.
2) Couldn't reproduce that one. Could you file that as a separate bug report with details to reproduce it?
3) It kind of mimics common office applications. In Google Docs for example it will tab to the next cell. Arrows down should take you out of the table though. If not then it's a bug that we should correct. Also file that separately so we can track that.

Thanks

**Comment by alecpl** [comment]
1) Alt+F10 works for me in Firefox an Chromium on Linux. Thanks for the hint. How do you think we could inform about this the user? Maybe short/small text in the toolbar?
3) Arrow down works, but again I didn't find it straightforward. Maybe again a short hint in the table toolbar?

Or all toolbars should have '?' button with such hints?

**Comment by alecpl** [comment]
Also, I'd like to know if/how it is possible to get access to Source code, Find and Replace, Undo, Redo functionality in Inlite mode. I guess these would need to be implemented out of the editor, right?

**Comment by spocke** [comment]
You can configure any tools in to for example the insert_toolbar. Check the docs here:
https://www.tinymce.com/docs/themes/inlite/

Hints could be provided by the implementation. A text box with quick keys separate from the editor. Or you could just make your own help button if you wanted to. :)
